setup/command/aws.go

```diff
@@ -23,14 +23,9 @@ var awsFlags = []cli.Flag{
 		Usage:  "secret key portion on an AWS key",
 		EnvVar: config.AWS_SECRET_ACCESS_KEY,
 	},
-	cli.StringFlag{
-		Name:   "aws-region",
-		Usage:  "AWS region",
-		EnvVar: config.AWS_REGION,
-	},
 }
 
-func (f *CommandFactory) newAWSProviderHelper(c *cli.Context) (*aws.Provider, error) {
+func (f *CommandFactory) newAWSProviderHelper(c *cli.Context, region string) (*aws.Provider, error) {
 	// use default credentials and region settings
```

Comment:
[nit] is this comment still relevant?

Reply:
It's not worded very well. It is relevant because it initially grabs default values and then overrides them later in this function. I'll reword this to make it clearer.